src/Sulu/Bundle/ContentBundle/.gitignore

```diff
@@ -3,6 +3,7 @@
 /composer.phar
 /composer.lock
 /.sass-cache/
+/Resources/app/cache
```

Comment:
"one day" also has the meaning "maybe we will do this, one day", but I
don't feel that strongly about it. But, seperate issue anyway.

On Thu, Apr 14, 2016 at 12:02:37AM -0700, Daniel Rotter wrote:
>    In [1]src/Sulu/Bundle/ContentBundle/.gitignore:
> 
>  > @@ -3,6 +3,7 @@
>  >  /composer.phar
>  >  /composer.lock
>  >  /.sass-cache/
>  > +/Resources/app/cache
> 
>    I would go even further and say we should remove the general rule from the
>    root .gitignore, and leave the bundle specific ignores in this file. This
>    way it would also work if we make a subtree split one day.
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu/pull/2278#discussion_r59670746
>    2. https://github.com/sulu/sulu/pull/2278/files/acb0590427d2149b1458810cceeb2e55b7cff2d7#r59670746
>    3. https://github.com/sulu/sulu/pull/2278/files/acb0590427d2149b1458810cceeb2e55b7cff2d7#r59670746